parm/ioda/bufr_atms.yaml

```diff
@@ -0,0 +1,176 @@
+observations:
+  - obs space:
+      name: bufr
+      obsdatain: "$(BUFR_in)"
+
+      exports:
+        variables:
+          # MetaData
+          timestamp:
+            datetime:
+              year: "*/YEAR"
+              month: "*/MNTH"
+              day: "*/DAYS"
+              hour: "*/HOUR"
+              minute: "*/MINU"
+              second: "*/SECO"
+
+          latitude:
+            query: "*/CLATH"
+
+          longitude:
+            query: "*/CLONH"
+
+          satelliteIdentifier:
+            query: "*/SAID"
+
+          satelliteInstrument:
+            query: "*/SIID"
+
+          # The type should be integer.
+          # However, UFO/CRTM requred this variable to be float
+          fieldOfViewNumber:
+            query: "*/FOVN"
+            type: float
+
+          heightOfStation:
+            query: "*/HMSL"
+
+          solarZenithAngle:
+            query: "*/SOZA"
+
+          solarAzimuthAngle:
+            query: "*/SOLAZI"
+
+          sensorZenithAngle:
+            query: "*/SAZA"
+
+          sensorAzimuthAngle:
+            query: "*/BEARAZ"
+
+          sensorScanAngle:
+            sensorScanAngle:
+              fieldOfViewNumber: "*/FOVN"
+              heightOfStation: "*/HMSL"
+              scanStart: -52.725
+              scanStep: 1.110
+
+          sensorChannelNumber:
+            query: "*/ATMSCH/CHNM"
+
+          # ObsValue
+          # Remapped Brightness Temperature
+          remappedBT:
+            remappedBrightnessTemperature:
+              fieldOfViewNumber: "*/FOVN"
+              sensorChannelNumber: "*/ATMSCH/CHNM"
+              brightnessTemperature: "*/ATMSCH/TMBR"
+              obsTime:
+                year: "*/YEAR"
+                month: "*/MNTH"
+                day: "*/DAYS"
+                hour: "*/HOUR"
+                minute: "*/MINU"
+                second: "*/SECO"
+
+        splits:
+          satId:
+            category:
+              variable: satelliteIdentifier
+              map:
+                _224: npp
```

Comment:
We identify different `satelliteIdentifiers` in the yaml but these different `satelliteIdentifiers` share many parameters in common.   What happens if a future atms instrument has slightly different parameters (e.g., `scanStart` or `scanStep`)?

Reply:
@emilyhcliu any thoughts on this?